ntbs-service/DataAccess/NotificationRepository.cs

```diff
@@ -31,9 +33,23 @@ public NotificationRepository(NtbsContext context)
             this.context = context;
         }
 
-        public async Task<IList<Notification>> GetNotificationsAsync() 
+        public async Task<IList<Notification>> GetRecentNotificationsAsync() 
         {
-            return await context.Notification.ToListAsync();
+            //Get TB service here
+            return await context.Notification
+            .Where(n => n.Episode.TBService.Name == "Ashford Hospital")
```

Comment:
placeholder TBservice while we don't have user logins

Reply:
Let's make it a parameter to this method though!
Also, a user could be a member of multiple TBs, so we should accommodate a list here!